<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PROJETO CLASSE PACIENTE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NOME : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PESO  :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>50</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ALTURA :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNome">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>170</width>
      <height>15</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAltura">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>170</width>
      <height>15</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPeso">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>170</width>
      <height>15</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalcular">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSaida">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
